samples/openapi3/client/petstore/typescript/builds/deno_object_params/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from "./http/http.ts";
+export * from "./auth/auth.ts";
+export * from "./models/all.ts";
+export { createConfiguration } from "./configuration.ts"
+export type { Configuration } from "./configuration.ts"
+export * from "./apis/exception.ts";
+export * from "./servers.ts";
+export { RequiredError } from "./apis/baseapi.ts";
+
+export type { PromiseMiddleware as Middleware } from './middleware.ts';
+export { type PetApiAddPetRequest, type PetApiDeletePetRequest, type PetApiFindPetsByStatusRequest, type PetApiFindPetsByTagsRequest, type PetApiGetPetByIdRequest, type PetApiUpdatePetRequest, type PetApiUpdatePetWithFormRequest, type PetApiUploadFileRequest, ObjectPetApi as PetApi,  type StoreApiDeleteOrderRequest, type StoreApiGetInventoryRequest, type StoreApiGetOrderByIdRequest, type StoreApiPlaceOrderRequest, ObjectStoreApi as StoreApi,  type UserApiCreateUserRequest, type UserApiCreateUsersWithArrayInputRequest, type UserApiCreateUsersWithListInputRequest, type UserApiDeleteUserRequest, type UserApiGetUserByNameRequest, type UserApiLoginUserRequest, type UserApiLogoutUserRequest, type UserApiUpdateUserRequest, ObjectUserApi as UserApi } from './types/ObjectParamAPI.ts';
```

Comment:
this is the meaningful change in the generated fixture. Please let me know if you prefer to introduce some newlines between the different imports/exports, which would make this a lot more readable, but also would increase the diff of this PR, because it would affect other samples as well.